{{cookiecutter.github_project_name}}/package.json

```diff
@@ -1,14 +1,75 @@
 {
-  "private": true,
+  "name": "{{ cookiecutter.npm_package_name }}",
+  "version": "{{ cookiecutter.npm_package_version }}",
+  "description": "{{ cookiecutter.project_short_description }}",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab extension",
+    "widgets"
+  ],
+  "homepage": "https://github.com/jupyter/{{ cookiecutter.npm_package_name }}",
+  "bugs": {
+    "url": "https://github.com/jupyter/{{ cookiecutter.npm_package_name }}/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": "",
+  "main": "lib/index.js",
+  "types": "./lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jupyter/{{ cookiecutter.npm_package_name }}"
+  },
   "scripts": {
-    "install": "lerna bootstrap --hoist",
-    "clean": "lerna run clean",
-    "build": "lerna run build",
-    "updated": "lerna updated",
-    "test": "lerna run test",
-    "publish": "npm run clean && npm run build && lerna publish -m \"Publish npm packages\""
+    "build": "npm run build:lib && npm run build:nbextension",
+    "build:labextension": "cd {{ cookiecutter.python_package_name }} && rimraf labextension && mkdirp labextension && cd labextension && npm pack ../..",
+    "build:lib": "tsc --project src",
+    "build:nbextension": "webpack",
+    "build:release": "npm run build:lib && npm run build:labextension && npm run build:nbextension",
+    "clean": "npm run clean:lib && npm run clean:nbextension",
+    "clean:lib": "rimraf lib",
+    "clean:labextension": "rimraf {{ cookiecutter.python_package_name }}/labextension",
+    "clean:nbextension": "rimraf {{ cookiecutter.python_package_name }}/nbextension/static/index.js",
+    "prepare": "npm run build:lib",
+    "test": "npm run test:firefox",
+    "test:chrome": "karma start --browsers=Chrome tests/karma.conf.js",
+    "test:debug": "karma start --browsers=Chrome --singleRun=false --debug=true tests/karma.conf.js",
+    "test:firefox": "karma start --browsers=Firefox tests/karma.conf.js",
+    "test:ie": "karma start --browsers=IE tests/karma.conf.js",
+    "watch": "npm-run-all -p watch:*",
+    "watch:lib": "tsc -w --project src",
+    "watch:nbextension": "webpack --watch"
+  },
+  "dependencies": {
+    "@jupyter-widgets/base": "^1.0.1",
+    "sort-package-json": "^1.7.1"
```

Comment:
Is this really needed as a run-time dependency?

Reply:
(i.e. would it not be OK as a dev dependency?)

Reply:
No, that snuck in there when I re-ordered locally.